src/main/java/hudson/remoting/Response.java

```diff
@@ -59,17 +60,17 @@
     @SuppressFBWarnings(value="SE_TRANSIENT_FIELD_NOT_RESTORED", justification="Only supposed to be defined on one side.")
     private transient long totalTime;
     @SuppressFBWarnings(value="SE_TRANSIENT_FIELD_NOT_RESTORED", justification="Bound after deserialization, in execute.")
-    private transient Request<?, ?> request;
+    private transient Request<RSP, ? extends Throwable> request;
```

Comment:
As in `Request`, we cannot use `EXC` here because the bound for the `Response` exception must be larger than those of the `Request` exception.